java/flight/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
+  license agreements. See the NOTICE file distributed with this work for additional
+  information regarding copyright ownership. The ASF licenses this file to
+  You under the Apache License, Version 2.0 (the "License"); you may not use
+  this file except in compliance with the License. You may obtain a copy of
+  the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
+  by applicable law or agreed to in writing, software distributed under the
+  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+  OF ANY KIND, either express or implied. See the License for the specific
+  language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>arrow-java-root</artifactId>
+        <groupId>org.apache.arrow</groupId>
+        <version>6.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Arrow Flight</name>
+    <artifactId>arrow-flight</artifactId>
+    <url>https://arrow.apache.org/blog/2019/10/13/introducing-arrow-flight/</url>
+
+    <packaging>pom</packaging>
+
+    <properties>
+        <dep.grpc.version>1.30.2</dep.grpc.version>
+        <dep.protobuf.version>3.17.3</dep.protobuf.version>
+    </properties>
+
+    <modules>
+        <module>flight-core</module>
+        <module>flight-grpc</module>
+        <module>flight-sql</module>
+    </modules>
+
+    <build>
```

Comment:
if adding `xolstice` default config for `protoc` and `grpc` artifacts at the top level, it might be best to also add this plugin at the top level too since the properties produced by the plugin are used at the top level (like ${os.detected.classifier}`